.eleventy.js

```diff
@@ -4,6 +4,10 @@ module.exports = config => {
     return collection.getAll().find(x => x.url === '/');
   });
 
+  config.addCollection('items', function(collection) {
+    return collection.getFilteredByGlob('./src/item/*.md');
```

Comment:
This is spot on! Maybe one thing that would be cool would be ordering by date, reversed?

Reply:
It makes a lot of sense, so that the feed provides the latest entries. I'll try to find a way 👍